tests/app/celery/test_service_callback_tasks.py

```diff
@@ -60,10 +60,15 @@ def test_send_delivery_status_to_service_post_https_request_to_service_with_encr
         "template_version": 1,
     }
 
+    # TODO why is 'completed_at' showing real time unlike everything else and does it matter?
```

Comment:
Does this mean `completed_at` isn't coming back as a string like `created_at` and `sent_at`? If so...I'm not sure about this one.

Reply:
We don't use 'completed_at' for anything as far as I know and we currently have the same problem with 'updated_at' that broke about 6-8 weeks ago.   So I will write a ticket to investigate those items, but I don't think this issue blocks the PR.